Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Gradle files\r\n.gradle/\r\nbuild/\r\n\r\n# Local configuration file (sdk path, etc)\r\nlocal.properties\r\n\r\n# Log/OS Files\r\n*.log\r\n\r\n# Android Studio generated files and folders\r\ncaptures/\r\n.externalNativeBuild/\r\n.cxx/\r\n*.apk\r\noutput.json\r\n\r\n# IntelliJ\r\n*.iml\r\n.idea/\r\nmisc.xml\r\ndeploymentTargetDropDown.xml\r\nrender.experimental.xml\r\n\r\n# Keystore files\r\n*.jks\r\n*.keystore\r\n\r\n# Google Services (e.g. APIs or Firebase)\r\ngoogle-services.json\r\n\r\n# Android Profiling\r\n*.hprof
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 00d7975d06ec36775ec6c03eee6d6e7e9b5b71b8)
+++ b/.gitignore	(date 1742659724127)
@@ -17,7 +17,6 @@
 
 # IntelliJ
 *.iml
-.idea/
 misc.xml
 deploymentTargetDropDown.xml
 render.experimental.xml
